kolibri/plugins/coach/assets/src/modules/classSummary/index.js

```diff
@@ -39,6 +39,7 @@ function defaultState() {
      *     question_sources: [{exercise_id, question_id}, ...],
      *     groups: [id, ...],
      *     data_model_version,
+     *     question_count,
```

Comment:
we're going to want to update the unit tests to have conforming data: 

* https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/coach/assets/src/modules/classSummary/__test__/sampleServerResponse.js
* https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/coach/assets/src/modules/classSummary/__test__/sampleState2.js

would you mind either updating those, or opening a new tech debt issue indicating that we should update them?

Reply:
Looks like I don't have a choice if I want tests to pass!